src/cascadia/TerminalSettingsEditor/Launch.xaml

```diff
@@ -70,16 +70,61 @@
                 </local:SettingContainer>
 
                 <!--  Default Terminal  -->
-                <!--  TODO GH#9463 - Complete hookup of Terminal UX to choose defapp.  -->
-                <!--
                 <local:SettingContainer x:Uid="Globals_DefaultTerminal">

```

Comment:
How can I do this? Binding of `x:Load` on the outside `local:SettingContainer` to a bool property that checks if we're on a supported OS platform?

Reply:
Basically, yeah. Here's what I'm thinking:

### XAML

Like you were saying, bind the setting container to a bool property.
```xml
<local:SettingContainer x:Load="{x:Bind IsDefTermAvailable}" />
```

### The bool property

Make sure you expose this in the IDL as a getter...
```
Boolean IsDefTermAvailable {get;};
```

Then in the .h/cpp add...
```cpp
bool IsDefTermAvailable()
{
   // do the check
}
```

We shouldn't have to worry about it being observable or anything like that because users can't enable/disable this feature without relaunching the app right?